src/util.cpp

```diff
@@ -1120,6 +1126,9 @@ std::string toString(const Decl *decl)
 {
     // Reference: https://clang.llvm.org/doxygen/classclang_1_1Decl.html
 
+    if (decl == nullptr)
+        return "Error: nullptr";
```

Comment:
Is returning this malformed string the best thing to do?  It seems like we should at least put some boundary around it like `[nullptr]` because `:` is used as a valid delimiter in the other strings.

Reply:
Sounds good to me. I didn't pay attention to the string pattern as I use this just for debugging.

Reply:
Should this throw some sort of message to `std::cerr` as per other warnings?

Reply:
Done: https://github.com/personalrobotics/chimera/pull/312/commits/ee496181137b95614226df7751d2ec26e741fa27